database/migrations/0001_01_01_000000_create_users_table.php

```diff
@@ -19,6 +18,7 @@ public function up(): void
             $table->string('password');
             $table->rememberToken();
             $table->timestamps();
+            $table->boolean('premium')->default(false);
```

Comment:
It looks like you're modifying a migration. This should mostly never be done. Once a migration is finalized (merged to master), you'll need to create a new migration to update the database. When modifying a migration, anyone who has already run the migration will have to start his database over in order to adopt the new migration (maybe he won't have to start over, but it usually won't be pretty to try to recover).